obal/data/roles/build_package/defaults/main.yaml

```diff
@@ -0,0 +1,4 @@
+---
+build_package_build_system: koji
+build_package_koji_command: 'koji'
```

Comment:
can this be set automatically depending on the value of `build_package_build_system`? Or maybe we can just use `build_package_build_system` directly? I guess it will be `koji`/`brew` where you want this distinction, and for all the other build systems, we'd probably just never load the `koji` related task files?

Reply:
build_package_build_system is to denote that we are using a koji-based workflow.

build_package_koji_command is to specify the exact koji command to use when interacting with the koji system (koji, brew, kkoji*, etc...).  This is a variable specific to using 'koji.yml'

* for a while, to denote between fedora's koji and katello's koji, I had a kkoji command on my system, which is supported by koji

Reply:
Right. I forgot about the kkoji stuff.